content/docs/searching.md

```diff
@@ -523,9 +523,9 @@ Anchors match a zero-length position in the line, rather than a particular chara
 
 * `\B` ⇒ Not a word boundary. It represents any location between two word characters or between two non-word characters.
 
-* `\A` or `\'` ⇒ The start of the matching string.
+* `\A` or `` \` `` ⇒ Matches the start of the file.
 
-* `\z` or `` \` `` <!-- ` --> ⇒ The end of the matching string.
+* `\z` or `\'` <!-- ` --> ⇒ Matches the end of the file.
```

Comment:
What's with the `<!-- ... -->` part of the `\z` line?
I realize that it is not a change here, just wondering its purpose.

Reply:
Unnecessary comment - cleaned up.

Reply:
As far as I know, markdown accepts HTML comments (just like it technically accepts other HTML syntax).  (Even Notepad++'s "Markdown (preinstalled)" recognizes HTML comments as comments.

However, those comments were there because apparently my Markdown UDL has a bug/deficiency in it, so it would extend the backtick-highlighting across many lines on the line with the escaped backtick inside the backticks.  I see that the "Markdown (preinstalled)" doesn't have that problem.  Interesting.  So yes, they were something I had just to make the formatting look right in Notepad++ while I was editing.  I guess I'm going to have to switch over to the builtin Markdown UDL rather than my own.